src/app/main/employees/employees-list/employees-list.component.ts

```diff
@@ -63,19 +65,17 @@ export class EmployeesListComponent implements OnInit {
   }
 
   filterEmployees(term: any, serchKey: string) {
-    term = (serchKey == 'by_department_name') ? term : term.target.value
-    if (term.trim() === serchKey) {
-      delete this.params[serchKey];
-    }
-    else {
-      this.params[serchKey] = term;
-    }
+    term = (serchKey == 'by_department_id') ? term : term.target.value
```

Comment:
Please rename it to **searchKey** and apply the change to all of it's occurrences.

Reply:
done